### ISSUE TITLE
copydoc on related function causes warning: @copybrief or @copydoc target not found

### ISSUE THREAD

**Issue by scott-zhong** [doxygen/doxygen#10648]
`copydoc` on  related function causes warning: `@copybrief` or `@copydoc` target not found. doxygen-1.9.7 does not generate the warning but doxygen-1.9.8 and later does, include 1.11.0 (trunk).

doxygen-1.9.8:

```
doxygen -g
doxygen Doxyfile
...
Computing tooltip texts...
C:/dev/doxygen-1.9.8.windows.x64.bin/test.h:23: warning: @copybrief or @copydoc target 'operator==(const Base&, const Base&)' not found
...
```

trunk:

```
doxygen -g
doxygen -x Doxyfile
# Difference with default Doxyfile 1.11.0 (5fa510c0901e7b679471473f88bdc50ec44a6bee*)
doxygen Doxyfile
...
Computing tooltip texts...
/mnt/c/dev/Ubuntu Latest GCC Release build artifacts/test.h:23: warning: @copybrief or @copydoc target 'operator==(const Base&, const Base&)' not found
...
```

Testcase file:
[testcase.zip](https://github.com/doxygen/doxygen/files/14240930/testcase.zip)

**Comment by albert-github** [comment]
I think this problem has the same base issue as the issue in #10640.

**Comment by doxygen** [comment]
@scott-zhong Please verify if the solution for #10640 also fixes this issue.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.11.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
